consoles/sshVirtsh.pm

```diff
@@ -136,8 +136,14 @@ sub _init_xml {
     }
 
     if (get_var('UEFI') and check_var('ARCH', 'x86_64') and !get_var('BIOS')) {
-        # These are known locations for openSUSE and Fedora (respectively).
-        my @known = ('/usr/share/qemu/ovmf-x86_64-ms.bin', '/usr/share/edk2.git/ovmf-x64/OVMF_CODE-pure-efi.fd');
+        # These are known locations: first is openSUSE, second is for
+        # the kraxel.org nightly packages, third is Fedora's edk2-ovmf
+        # package.
+        my @known = (
+            '/usr/share/qemu/ovmf-x86_64-ms.bin',
+            '/usr/share/edk2.git/ovmf-x64/OVMF_CODE-pure-efi.fd',
+            '/usr/share/edk2/ovmf/OVMF_CODE.fd',
```

Comment:
Probably github is broken and just shows this incorrectly. I wouldn't expect you from simply duplicating this even more ;-)

[DRY](https://en.wikipedia.org/wiki/Don't_repeat_yourself). You are calling the same code multiple times, please [extract method](http://refactoring.com/catalog/extractMethod.html).